server/airline_router.go

```diff
@@ -13,4 +13,5 @@ func (srv *AppServer) AirlineRouter(db *gorm.DB) {
 	
 	srv.router.GET("/airline/", airlineController.HandleGetAirline)
```

Comment:
The Controller method here should be plural - since the returned data is logically an array of airline records

Reply:
okay